### DISCUSSION TITLE
Noob here looking for tuts

### DISCUSSION THREAD

**Discussion by etenebris** [paypal/paypal-ios#187] [Q&A] [answered]
Hi, I'm a noob.  I have an app on the app store and I'm adding functionality to it over time.  Call me an intermediate noob.  Anyhow, are there any tutorials for your SDK and how to add it to an existing xcode project?  I've found one on yt called ' PayPal Payment Gateway Integration With iOS Swift | XCODE 10.2 | Swift 5' by 'Swift Series' but that is 3 years old at this point and uses a deprecated pod...

**Comment by scannillo** [comment, marked as answer]
Please see our README and linked[ developer documentation](https://github.com/paypal/paypal-ios#modules).

**Reply by etenebris** [reply]
So I'm trying to run the Demo and I'm getting the following build fail error:

No such module 'CorePayments'

on the following swift class:
@testable import CorePayments.                          x No such module 'CorePayments'

class MockGraphQLClient: GraphQLClient

I installed the package dependency from this link with all options selected which did not resolve the issue:
https://github.com/paypal/paypal-ios

Otherwise, I have made a lot progress getting the Ppal developer and sandbox items set up and installing the pod 'PayPal/CardPayments'.  Again, I am a noooob doing my best to wrastle with adding ppal to my app.  Thank you in advance for any help you can render this old dog learning new tricks.
(xcode 14.2, Monterey 12.6.5)

**Reply by etenebris** [reply]
Okay, I take it back... after simply installing the package dependency to my project (and the various pods) I get a build fail with no specific reference:

x Multiple commands produce '/Users/.../Library/Developer/Xcode/DerivedData/ETE-bmedizbarzhjbrburjteycxcvizr/Build/Products/Debug-iphonesimulator/ETE.app/Frameworks/PayPalCheckout.framework'

Removing the package dependency eliminates the build fail message...

**Comment by scannillo** [comment]
Please instead open a GitHub Issue using[ this form](https://github.com/paypal/paypal-ios/issues/new/choose) with steps to replicate, so we can appropriately triage and track your issue!
